### ISSUE TITLE
Wishlist: Remove cond's default else-is-void clause and replace with a default else-is-error.

### ISSUE THREAD

**Issue by spdegabrielle** [racket/racket2-rfcs#39]
Remove cond's default else-is-void clause and replace with a default else-is-error.
-taken from Wishlist of backwards incompatible things for a future Racket2. #33

**Comment by sorawee** [comment]
How is this different from #38?

**Comment by AlexKnauth** [comment]
I agree, replacing fallthrough behavior (when there's no else and all the conditions are false) from void to error. There have been so many bugs written by myself and other people that would have been caught much sooner if an error were the default fallthrough behavior

**Comment by sorawee** [comment]
`case` too?

**Comment by mflatt** [comment]
#527
